src/services/zapper.ts

```diff
@@ -135,7 +135,7 @@ export class ZapperService extends Service {
    * @returns gas prices
    */
   async gas(): Promise<GasPrice> {
-    const url = "https://api.zapper.fi/v1/gas-price";
+    const url = "https://api.zapper.fi/v2/gas-prices";
```

Comment:
v1 appears it wasn't working

<img width="2352" alt="Screen Shot 2022-04-26 at 8 42 25 pm" src="https://user-images.githubusercontent.com/78794805/165282901-2772a8b6-c271-4b52-bdec-bab6c45a1d41.png">

<img width="2352" alt="Screen Shot 2022-04-26 at 8 42 50 pm" src="https://user-images.githubusercontent.com/78794805/165282975-50a2fceb-3d0c-4f18-a0c0-ece6fec749b3.png">

Reply:
yeah, noticed this one is no longer use internally or by the frontend... We may want to maintain it though, since might be useful for some use cases.

should we add the `eip1559` param? seems its required in their docs

Reply:
I've added that to the types, it's here https://github.com/yearn/yearn-sdk/pull/284/files#diff-f0f290a5b03c672c900f63446a8be3076d2dc101a961cc6ef45f678313d0f866R7

Reply:
was added for the response, but not for the request payload. should we add it there too?
![image](https://user-images.githubusercontent.com/18649057/167675870-10127c1f-18b8-47a6-930b-9421da0fa198.png)

Reply:
Ah, interesting, missed this because the request still goes without the "required" param.

Added with https://github.com/yearn/yearn-sdk/pull/284/commits/9b7e1237c6f209181662ca6b7e0a93a184139728 setting it to `false`

When set to `false`
```json
{
  "eip1559": false,
  "standard": 30,
  "fast": 30,
  "instant": 30
}
```

when set to `true`
```json
{
  "eip1559": true,
  "standard": {
    "baseFeePerGas": 34,
    "maxPriorityFeePerGas": 1,
    "maxFeePerGas": 52
  },
  "fast": {
    "baseFeePerGas": 34,
    "maxPriorityFeePerGas": 2,
    "maxFeePerGas": 52
  },
  "instant": {
    "baseFeePerGas": 34,
    "maxPriorityFeePerGas": 2,
    "maxFeePerGas": 53
  }
}
```